apps/wpcom-block-editor/src/wpcom/features/redirect-onboarding-user-after-publishing-post.js

```diff
@@ -25,6 +25,7 @@ export function RedirectOnboardingUserAfterPublishingPost() {
 		hasStartWritingFlowQueryArg &&
 		'full' === launchpadScreen &&
 		currentPostType === 'post';
+	const postFlowUrl = getQueryArg( window.location.search, 'postFlowUrl' );
```

Comment:
Non-block nit: we could also call this `postPublishUrl`, but I don't think it's worth much rework.